ConsoleApp1/Vote.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Vote : Base
+    {
+        public static List<Vote> votes  { get; set; } = new List<Vote>()
```

Comment:
Create a class VoteSystem where you will store all votes, it will have Show, Add votes

Reply:
added